Add support for Firebase Remote Config (#2622)
firebase/firebase-tools: CHANGELOG.md

- Replaces `tslint` with `eslint` as the default linter in new Cloud Functions for Firebase directories. (#1663)
- Fixes bug where scheduled Cloud Functions for Firebase deploys unexpectedly succeeded with invalid Cloud Scheduler configurations. (#2609)

> - Add support for Firebase Remote Config (see README for list of commands)

Reviewer: Judging based on past changelog entries, I think this should probably say something along the lines of:

```suggestion
- Adds Firebase Remote Config commands: `remoteconfig:get`, `remoteconfig:versions:list`, `remoteconfig:rollback`.
- Adds a `remoteconfig` target for `firebase deploy`.
```

Author: Thanks for the suggestion! Updated :)
